src/app/item-card/item-card.component.ts

```diff
@@ -1,5 +1,6 @@
 import { Component, Input, OnInit } from '@angular/core';
 import { WindowService } from '../shared/services/window.service';
+import { RatingModule } from 'ngx-rating';
```

Comment:
Is this meant to be here, I deleted it from the file and it didn't cause any errors. Should this import be deleted?